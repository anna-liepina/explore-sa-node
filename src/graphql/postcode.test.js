describe('GraphQL: Postcode', () => {
    describe('query', () => {
        it(`fetching postcodes by exact postcodes`, async () => {
            const { data: { postcodeSearch: data } } = await query({
                query: `
                {
                    postcodeSearch(pattern: "E20 1AA") {
                        postcode
                        lat
                        lng
                        url
                    }
                }`
            });

            expect(data).toHaveLength(1);
            expect(data).toMatchSnapshot();
        });

        it(`fetching postcodes by wildcard`, async () => {
            const { data: { postcodeSearch: data } } = await query({
                query: `
                {
                    postcodeSearch(pattern: "E20") {
                        postcode
                        lat
                        lng
                        url
                    }
                }`
            });

            expect(data).toMatchSnapshot();
        });
    });
});
