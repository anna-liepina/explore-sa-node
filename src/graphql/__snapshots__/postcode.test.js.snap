// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GraphQL: Postcode mutation xxx 1`] = `undefined`;

exports[`GraphQL: Postcode query xxx 1`] = `undefined`;
